server/controllers/auth/reset_password.coffee

```diff
@@ -3,13 +3,13 @@ __ = require('config').universalPath
 _ = __.require 'builders', 'utils'
 error_ = __.require 'lib', 'error/error'
 user_ = __.require 'controllers', 'user/lib/user'
-testEmail = __.require('models', 'tests/user').email
+validate = __.require('models', 'validations/user').email
 # pw_ = __.require('lib', 'crypto').passwords
 
 module.exports = (req, res, next)->
   { email } = req.body
   unless email? then return error_.bundleMissingBody req, res, 'email'
-  unless testEmail email
+  unless validate email
```

Comment:
can't the function keep a more precise name like `validateEmail`?

Reply:
`isValid`?